api/provider/aws/load_balancer_update.go

```diff
@@ -82,6 +82,13 @@ func (l *LoadBalancerProvider) Update(loadBalancerID string, req models.UpdateLo
 		}
 	}
 
+	if req.IdleTimeout != 0 {
```

Comment:
update request fields should be pointers, so this will need to become `if req.IdleTimeout != nil`